core/services/synchronization/stats_pusher.go

```diff
@@ -26,8 +26,18 @@ var (
 	})
 )
 
-// StatsPusher polls for events and pushes them via a WebSocketClient
-type StatsPusher struct {
+//go:generate mockery -name StatsPusher -output ../../internal/mocks/ -case=underscore
```

Comment:
Should we leave a little comment here saying that this is to generate the `StatsPusher` mock? I think something like this belongs in a README of some sort more than anything.

Reply:
https://github.com/smartcontractkit/chainlink/pull/2229#pullrequestreview-349730923